src/test/java/ToDoTest.java

```diff
@@ -0,0 +1,23 @@
+import duke.task.ToDo;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ToDoTest {
+    @Test
+    public void testCheckDescription() {
+        String desc = "Checking ToDo Task";
+        ToDo todo = new ToDo(desc);
+        assertEquals(desc, todo.getDesc());
+    }
+
+    @Test
+    public void testCheckDone() {
+        String icon = "O";
+        String desc = "Checking ToDo icon";
+        ToDo todo = new ToDo(desc);
+        todo.markAsDone();
+        assertEquals(icon, todo.getStatusIcon());
+    }
+}
```

Comment:
Overall, i like how clean the code is. Each file has a specific function and the code is not cluttered in a single file. I actually learnt quite a bit from how you wrote your codes, and how you organise them! Great work! I am actually looking forward to seeing more of your coding style in the future!